src/server_manager/web_app/ui_components/outline-gcp-create-server-app.ts

```diff
@@ -364,7 +364,12 @@ export class GcpCreateServerApp extends LitElement {
     // `this.regionPicker` is null after `this.currentPage`, and is only populated
     // asynchronously.
     this.regionPicker = this.shadowRoot.querySelector('#regionPicker') as OutlineRegionPicker;
-    this.regionPicker.options = filterOptions(zoneOptions);
+    const displayOptions = filterOptions(zoneOptions);
```

Comment:
This is not yet RegionPickerOption, so let's not call it `displayOptions`.

Also, let's not modify the return value of listLocations. That could potentially affect future calls. You need to read that function to make sure things are not breaking. Instead, make a copy as part of the conversion to RegionPickerOption.

Reply:
Done